index.html

```diff
@@ -3126,7 +3125,7 @@ <h3>Exploration Mechanisms</h3>
         "<a>exploration</a>" mechanisms that provide actual access to metadata.
         </p><p>
         The first and simplest <a>exploration</a> mechanism defines how to fetch a <a>TD</a>
-        directly from the device itself. This supports ad-hoc peer-to-peer discovery.
+        directly from a simple web resource or from the device itself. This supports ad-hoc peer-to-peer discovery.
```

Comment:
note: this change is not directly related to the "make section informative" but is an editorial correction.

Reply:
I decided to revert this change to not trigger discussion about "unrelated changes", although I'll probably want to bring it back in later during the CR period as an editorial change.